build.gradle

```diff
@@ -127,6 +127,14 @@ allprojects {
         exclude group: 'ch.qos.logback', module: 'logback-classic'
         exclude group: 'ch.qos.logback', module: 'logback-core'
     }
+    normalization {
+        runtimeClasspath {
+            properties {
```

Comment:
so to confirm we want to ignore `version.date` in all files as well as the entire contents of all `views.properties` files?